css/css-lists/counter-reset-reversed-list-item.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Lists: counter-reset reversed(list-item)</title>
+<link rel="help" href="https://drafts.csswg.org/css-lists/#counter-reset">
+<link rel="match" href="counter-7-ref.html">
+<style>
+  .li { display: list-item; }
+  .reset-reversed { counter-reset: reversed(list-item) }
```

Comment:
Looking at the spec again, the rule for calculating the default value here is wrong - it says:

> the user agent must dynamically calculate the initial value at layout-time to be the number of elements and pseudo-elements that instantiate the same counter in the same scope, plus 1.

But you only instantiate a counter when you're creating a new one. You want to instead count, I guess, the number of elements that affect the counter? Or perhaps pretend it starts at -1, accumulate all the increments it receives, then start it at the negation of the final result (so it'll end on 1)?

Reply:
Spec fixed in https://github.com/w3c/csswg-drafts/pull/6297